website/docs/metadata.md

```diff
@@ -134,7 +134,7 @@ from different engines as shown below:
 |----------------------------------------------------------------------------------|------------------------|-------------------------------------------------------------------------------------------------------------------------------|
 | <ul><li>Spark DataSource</li><li>Spark SQL</li><li>Strucured Streaming</li></ul> | hoodie.metadata.enable | When set to `true` enables use of the spark file index implementation for Hudi, that speeds up listing of large tables.<br /> |
 |Presto| [hudi.metadata-table-enabled](https://prestodb.io/docs/current/connector/hudi.html)             | When set to `true` fetches the list of file names and sizes from Hudi’s metadata table rather than storage.                   |
-|Trino| [hudi.metadata-enabled](https://trino.io/docs/current/connector/hudi.html#general-configuration) | When set to `true` fetches the list of file names and sizes from metadata rather than storage.                                |
+|Trino| N/A | Support for reading from the metadata table [has been dropped in Trino 419](https://github.com/trinodb/trino/pull/17392). |
```

Comment:
Can you please link the JIRA ticket where we are working on the fix - https://issues.apache.org/jira/browse/HUDI-7020

Reply:
Thanks @codope, I wasn't aware of that ticket. Linked to it instead and pushed again.